src/server/express.js

```diff
@@ -19,7 +19,12 @@ export default function() {
 
   app.get('*', (req, res) => {
     const acceptsLanguages = req.acceptsLanguages(config.appLocales);
-    render(req, res, acceptsLanguages || config.defaultLocale)
+    const userState = {
+      i18n: {
+        locales: acceptsLanguages || config.defaultLocale
```

Comment:
This now have an impact on a state returned to a user as previously, we were doing nothing with the `acceptsLanguages` variable. Obviously, no changes at all, as we preload messages inside an `initialState` based on `locales` key during the first require. One solution would be to make initialState return a function that optionally takes an object to merge the state with. This will allow loading custom messages.